packages/smart-contracts/hardhat.config.ts

```diff
@@ -77,6 +74,16 @@ export default {
       url: process.env.WEB3_PROVIDER_URL || 'https://rpcapi.fantom.network',
       chainId: 250,
       accounts,
+    },
+	arbitrum: {
```

Comment:
Try applying prettier on this file manually as it is not applied by `yarn run format:check` because this file is not in a `src` directory. I will create a PR to fix the fact that it is not checked.

Reply:
cf https://github.com/RequestNetwork/requestNetwork/pull/732

Reply:
Done